experimental/PlaneCurve/ParaPlaneCurves.jl

```diff
@@ -140,6 +258,40 @@ function rat_normal_curve_It_Proj_Odd(C::ProjCurve)
     return gens(ideal(R, J))
 end
 
+@doc Markdown.doc"""
+    rat_normal_curve_It_Proj_Even(C::ProjCurve)
+
+Return a vector `PHI` defining an isomorphic projection of `C` to `PP^1`.
```

Comment:
What is PP^1? A projective line? Perhaps this?
```suggestion
Return a vector `PHI` defining an isomorphic projection of `C` to the projective line $\mathbb{P}^1`.
```